static/js/hotspots.js

```diff
@@ -58,6 +58,14 @@ const HOTSPOT_LOCATIONS = new Map([
 // popover illustration url(s)
 const WHALE = "/static/images/hotspots/whale.svg";
 
+const meta = {
+    opened_hotspot_name: null,
+};
+
+export function set_opened_hotspot_name(name_or_null) {
+    meta.opened_hotspot_name = name_or_null;
+}
```

Comment:
I think it probably doesn't make sense to export this interface to other modules -- it's kinda an abstraction violation. I think a good option would be to create a preparatory refactoring commit that moves the hotspots click handlers to be defined in the `initialize` function in this module, rather than in the catch-all `click_handlers.js`; and then once that's done, we can have a much cleaner abstraction boundary for this module. Thoughts?

Reply:
That sounds reasonable to me! I chose this path largely to keep the changes made on the lighter side given that I'm new to the codebase, but I agree that moving the click handler into `hotspots.js` is a cleaner/more easily understood way of handling this.

I'll get those commits in and we can see how it looks from there.